entrypoint.sh

```diff
@@ -23,12 +31,11 @@ echo "Start: Sonar Scan"
 sh -c "/scripts/coverage_scan.sh"
 echo "End: Sonar Scan"
 
-echo "Container Push: $INPUT_CONTAINER_PUSH_ENABLED"
-if [ "$INPUT_CONTAINER_PUSH_ENABLED" = 'true' ]; then
```

Comment:
Any specific reason if condition here is removed, If this is because there won't be any application which is not containerized in our organization then i am fine with change. This flag was added under the assumption to support non containerized applications and they just want to do nuget push.

Reply:
Fixed.